sport/test/package.scala

```diff
@@ -36,13 +36,26 @@ class SportTestSuite extends Suites (
   new MatchFeatureTest,
   new ResultsFeatureTest,
   new rugby.model.MatchParserTest
-) with SingleServerSuite with FootballTestData with BeforeAndAfterAll with WithTestWsClient {
-
+) with SingleServerSuite with BeforeAndAfterAll with WithTestWsClient {
   override lazy val port: Int = new HealthCheck(wsClient).testPort
+}
+
+trait WithTestFootballClient {
+  self: WithTestFootballClient with BeforeAndAfterAll with WithTestWsClient =>
```

Comment:
`self: BeforeAndAfterAll with WithTestWsClient =>`
You are specifying the type twice here I believe